pkg/kubelet/apis/kubeletconfig/types.go

```diff
@@ -213,7 +213,8 @@ type KubeletConfiguration struct {
 	// PodPidsLimit is the maximum number of pids in any pod.
 	PodPidsLimit int64
 	// ResolverConfig is the resolver configuration file used as the basis
-	// for the container DNS resolution configuration.
+	// for the container DNS resolution configuration, set it to "" to prevent
+	// Pods from inheriting DNS configuration from node.
 	ResolverConfig string
```

Comment:
if this isn't a pointer, roundtripping loses the distinction between nil and ""

Reply:
would pointing this at an empty file accomplish the same thing?

Reply:
Hi, the command-line flag works fines now, it could distinguish between empty and "". I think we should fix the config file option ResolverConfig's behavior to match with the command-line flag.

I guess that allow pointing this at an empty file would be a little confusing and inconvenient.